src/net/sourceforge/kolmafia/textui/command/HeistCommand.java

```diff
@@ -0,0 +1,82 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import net.sourceforge.kolmafia.FamiliarData;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.KoLConstants.MafiaState;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.RequestLogger;
+import net.sourceforge.kolmafia.objectpool.FamiliarPool;
+import net.sourceforge.kolmafia.persistence.ItemDatabase;
+import net.sourceforge.kolmafia.session.HeistManager;
+
+public class HeistCommand extends AbstractCommand {
+  public HeistCommand() {
+    this.usage = " [ITEM] - display all heistable items, or heist a specific item";
+  }
+
+  @Override
+  public void run(final String cmd, String parameter) {
+    FamiliarData current = KoLCharacter.getFamiliar();
+    if (current == null || current.getId() != FamiliarPool.CAT_BURGLAR) {
+      KoLmafia.updateDisplay("You need to take your Cat Burglar with you");
+      return;
+    }
+
+    parameter = parameter.trim();
+
+    if (parameter.equals("")) {
+      // show all items
+      StringBuilder output = new StringBuilder();
+
+      var heistManager = heistManager();
```

Comment:
Are we using `var` in non-test code yet? I've been avoiding it but I'm not sure if being so careful is actually required.

Reply:
I like it too, we should allow it.